packages/common/src/utils/formatters/formatters.ts

```diff
@@ -8,6 +8,7 @@ export const Formatter = {
     `${withParens ? '(' : ''}${(tax ?? 0).toFixed(2)}%${withParens ? ')' : ''}`,
   date: (date: Date): string =>
     `${date.getDate()}/${date.getMonth() + 1}/${date.getFullYear()}`,
+  localisedDate: (date: Date): string => format(date, 'P'),
```

Comment:
I couldn't see a localised date function anywhere else, so this would probably be useful?

Reply:
There's a `useFormatDate` hook in there, but it seems to be returning US-style dates instead of localised ones. You can pass a DateTimeFormatOptions object to it to specify formatting, but we want the date to automatically show in the user's local format regardless of where they are. It seems the `date-fns` functions is the only way to (easily) auto-format in local date format. 

Perhaps the `useFormatDate` function needs to be updated to use this? I'll make a new issue to figure it out and we can address it next week?

Reply:
New issue: https://github.com/openmsupply/openmsupply-client/issues/1145 
Let's discuss next week.